packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-hand.tsx

```diff
@@ -6,16 +6,16 @@ import { teamsIconClassNames } from '../teamsIconClassNames'
 export default {
   icon: ({ classes }) => (
     <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
-      <g transform="translate(-638.000000, -381.000000)">
+      <g transform="matrix(0.78936249,0,0,0.78936249,-500.24307,-297.37694)">
```

Comment:
Can you run these SVGs through SVGOMG?
https://jakearchibald.github.io/svgomg/

This should get rid of the extra G tags and the transform.

Reply:
It doesn't reduce the size of the SVGs by much, but the more simple it can be, the better. :)